### DISCUSSION TITLE
mkdocs-material is now included in cookiecutter template

### DISCUSSION THREAD

**Discussion by zillionare** [squidfunk/mkdocs-material#2672] [Show and tell]
I'm huge fan of mkdocs/mkdocs-material, and made a cookiecutter template in which mkdocs/mkdocs-material is the documentation builder. You may consider this template (and also a standalone tool called "python project wizard") if you'd like to use mkdocs too.

The template is [here](https://github.com/zillionare/cookiecutter-pypackage)

**Comment by squidfunk** [comment]
Thanks for sharing!
